src/System.Private.CoreLib/src/System/Collections/Generic/EqualityComparer.cs

```diff
@@ -27,8 +27,10 @@ private static EqualityComparer<T> Create()
             // instantiation-specific implementation.
             throw new NotSupportedException();
 #else
-            // CORERT: TODO: https://github.com/dotnet/corert/issues/763
-            return (_default = new DefaultEqualityComparer<T>());
```

Comment:
`DefaultComparer/DefaultEqualityComparer` can be deleted now (or moved under `#if false` if we want to keep it around for testing purposes).